backends/fs/client.go

```diff
@@ -0,0 +1,63 @@
+package fs
+
+import (
+	"os"
+	"io/ioutil"
+	"path"
+	"path/filepath"
+	"strings"
+)
+
+type Client struct {
+	rootPath    string
+	maxFileSize int64
+}
+
+func NewFsClient() (*Client, error) {
+	// TODO: how to provide rootPath in order to pass tests?
+	rootPath := os.Getenv("BACKEND_FS_ROOTPATH")
+	// TODO: limit in size (for now 10mb), maybe retrieve from args or etcd default value or whatever.
+	maxFileSize := int64(10485760)
+	return &Client{rootPath, maxFileSize}, nil
+}
+
+func (c *Client) GetValues(keys []string) (map[string]string, error) {
+	vars := make(map[string]string)
+	for _, key := range keys {
+		keyPath := path.Join(c.rootPath, key)
+		if _, err := os.Stat(keyPath); os.IsNotExist(err) {
+			return vars, err
+		}
+
+		walk := func(fpath string, fi os.FileInfo, err error) error {
+			if (err != nil) {
```

Comment:
From https://golang.org/doc/effective_go.html#control-structures, parentheses in control structures are not allowed. You should've seen compile errors IIRC